tutorials/global_illumination.md

```diff
@@ -0,0 +1,419 @@
+\page global_illumination Global Illumination
+
+This tutorial showcases how to enable real-time global illumination in Gazebo.
+
+<div style="text-align:center;">
+    \image html files/global_illumination/global_illumination.gif width=60%
+</div>
+
+## About global illumination
+
+Global illumination (GI) techniques account for illumination from indirect light reflections. When GI is enabled, objects not only receive direct light from light sources, but also from indirect light bouncing off of other surfaces, ensuring nice reflections and soft shadows.
+
+Gazebo supports two GI methods through the Ogre2 rendering engine, Voxel Cone Tracing (VCT) and Cascaded Image Voxel Cone Tracing (CI VCT).
+
+### Voxel Cone Tracing
+
+A scene rendered with VCT is reliable and good quality, and thus is the best choice for most scenes.
+
+VCT parameters include:
+* **Resolution**
+* **Octant count**
+* **Bounce count**
+* **High quality**
+* **Anisotropic**
+* **Thin wall counter**
+* **Conserve memory**
+* **Debug visualization mode**
+
+### Cascaded Image Voxel Cone Tracing
+
+Compared to VCT, a scene rendered with CI VCT is slightly lower resolution, but the scene can be re-voxelized every frame, so it’s much faster when dealing with very large scenes. However, it is more memory-intensive than VCT.
+
+CI VCT parameters include:
+* **Bounce count**
+* **High quality**
+* **Anisotropic**
+* **Debug visualization mode**
+* **Cascade**
+    * **Resolution**
+    * **Octant count**
+    * **Thin wall counter**
+    * **Area half size**
+
+## Enabling global illumination in Gazebo
+
+During simulation, the GUI and the camera sensor view are two different scenes. The GUI scene is rendered on the frontend client process, and the camera sensor scene is rendered on the backend server process. Thus, GI can be enabled on either or both of these processes.
+
+> **Note:** GI solutions require hardware that uses DX11 or GL4. <br>
```

Comment:
we haven't really tested it on DX11 before so let's not mention it for now until we verify it.

```suggestion
> **Note:** GI solutions require hardware that uses OpenGL4. <br>
```